config/locales/server.en.yml

```diff
@@ -1565,6 +1565,7 @@ en:
     unsubscribe_via_email_footer: "Attach an unsubscribe via email mailto: link to the footer of sent emails"
 
     delete_email_logs_after_days: "Delete email logs after (N) days. 0 to keep indefinitely"
+    delete_post_reply_keys_after_days: "Delete post_reply_keys after (N) days. 0 to keep indefinitely"
```

Comment:
I would actually name that setting something like `allow_email_reply_for_days`. 

That is more user friendly than the `reply keys` technicality.

Reply:
> That is more user friendly than the reply keys technicality. 
Good idea :+1:  Let me update the name for the site setting.